modules/doc/content/framework_development/sanity_checking.md

```diff
@@ -55,3 +55,56 @@ pair of parameters used in testing can be used to disable the kernel coverage ch
   solve = false
 []
 ```
+
+Regarding simulations like additive manufacturing or immersed boundary simulations, users often work with a complete mesh where certain regions (referred to as *inactive domains*) should not be solved.
```

Comment:
By default, MOOSE assumes the computational domain to be the entire mesh. However, some simulations set up using subdomain modifiers or lower-dimensional blocks (such as mortar contact) define computational domain of the underlying problem on a subset of the mesh.